lib/matplotlib/rcsetup.py

```diff
@@ -1115,7 +1115,7 @@ def _validate_linestyle(ls):
     'text.color':          ['black', validate_color],
     'text.usetex':         [False, validate_bool],
     'text.latex.unicode':  [True, validate_bool],
-    'text.latex.preamble': [[''], validate_stringlist],
+    'text.latex.preamble': [[], validate_stringlist],
```

Comment:
This change appears unrelated.

Reply:
That's because `matplotlib.rcsetup.validate_stringlist([""]) == []` so the code I deleted complained that the validator was not idempotent here.